ro_help/hub/models.py

```diff
@@ -260,6 +260,14 @@ def get_absolute_url(self):
     def get_funders(self):
         return self.payment_orders.filter(success=True)
 
+    def get_total_funded(self):
+        no_funders = self.payment_orders.filter(success=True).count()
+        if no_funders:
```

Comment:
nit: can we change the variable name to `num_founders` or simply `founders`? In English, it reads as though the variable is called "no funders", as opposed to "number of funders".